DifferentiationInterface/README.md

```diff
@@ -48,6 +48,10 @@ We support all of the backends defined by [ADTypes.jl](https://github.com/SciML/
 - [Tracker.jl](https://github.com/FluxML/Tracker.jl)
 - [Zygote.jl](https://github.com/FluxML/Zygote.jl)
 
+Note that in some cases, using DifferentiationInterface.jl will be slower than calling backend-specific utilities.
+This is mostly true for Enzyme.jl, whose handling of activities and multiple arguments unlocks additional performance compared to our simple API.
```

Comment:
Also add closures specifically and perhaps something along the lines of "for users who want to differentiate such code with Enzyme, consider using Enzyme.autodiff / Enzyme.gradient / Enzyme.jacobian / Enzyme.hvp directly"

Reply:
Okay for closures, but to recommend Enzyme syntactic sugar I'd love for it to pass all my DI tests. Last time I checked, there were still cases where the Enzyme version is wrong, eg https://github.com/EnzymeAD/Enzyme.jl/issues/1391.

Reply:
Oh sorry missed that issue, but then perhaps just say autodiff / gradient / hvp directly then?

My guess for your issue looking at it is that it does have all the correct terms but per row major col major stuff it’s row major in one part col in other (as Julia is col major iirc)

Reply:
Yeah that might be it. My point is: as soon as there is an operator available in Enzyme, I try to use it in DI too instead of reinventing the wheel. Assuming this Jacobian issue gets fixed and I also integrate your new HVP operator without troubles, I'm okay with adding a pointer to the Enzyme syntax 😊 sound good?

Reply:
oh also we should add something like "Some functions are differentiable with Enzyme directly, but not with DI". Since this one feels more critical to know, maybe we just write "consider using Enzyme directly" and update this to add gradient/jacobian/hvp/etc once those are in?